sentences/pt/light_HassTurnOff.yaml

```diff
@@ -3,14 +3,14 @@ intents:
   HassTurnOff:
     data:
       - sentences:
-          - "<desliga> [(todas | todos)] [(o | os | a | as)] <luz> <na_zona>"
+          - "<desliga> [tod[a|o]s] [(o[s]| a[s])] <luz> <na_zona>"
         slots:
           domain: "light"
 
       # Desligar luzes na mesma divisão que um dispositivo de voz
       - sentences:
-          - "<desliga> [(o | os | a | as)] <luz>"
-          - "<desliga> [(todas | todos)] [(o | os | a | as)] <luz> aqui"
+          - "<desliga> [(o[s]| a[s])] <luz>"
```

Comment:
```suggestion
          - "<desliga> [(o[s]|a[s])] <luz>"
```

Reply:
Thanks a lot @abmantis, I also added the changes you proposed.